File: Data/Solist.cs
using System.ComponentModel.DataAnnotations;

namespace MaestroNotes.Data
{
    public class Solist
    {
        [Key]
        public int Id { get; set; }
        [MaxLength(50)]
        public string Vorname { get; set; } = "";

        [MaxLength(50)]
        public string Name { get; set; } = "";
        public DateTime? Born { get; set; }
        [MaxLength(1000)]
        public string? Note { get; set; }

        public DateTime? Died { get; set; }
        // ALTER TABLE Solisten ADD Died DATETIME NULL;

        public List<MusicRecord> MusicRecords { get; set; } = new();
    }
}

File: php-app/app/Models/Solist.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Solist extends Model
{
    use HasFactory;

    protected $table = 'solisten';

    protected $fillable = [
        'vorname',
        'name',
        'born',
        'note',
    ];

    protected $casts = [
        'born' => 'date',
    ];

    public function musicRecords()
    {
        return $this->belongsToMany(MusicRecord::class, 'music_record_solist');
    }
}
